Lib/unittest/test/testmock/testwith.py

```diff
@@ -126,6 +126,21 @@ def test_dict_context_manager(self):
 
         self.assertEqual(foo, {})
 
+    def test_double_patch_instance_method(self):
+        # See http://bugs.python.org/issue26704
```

Comment:
Any chance you could lose this? I know there are plenty of these links already in the source, but they're not going to mean much in 10 years time when we're on a different tracker...